### PR TITLE
ADOPTERS: add meiyapico as sample user

### PR DESCRIPTION
**Pull request by complone**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Comment by zwzhang0107** [comment]
Welcome!

File: ADOPTERS.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ free to get in touch.
...
 | [360](https://www.360.com)             | [@liuming](https://github.com/lucming)                                               | Stable   | 1. Machines running some middleware procs they can limit the resources they use and we will use the idle resources to run some online/offline pods to increase resource utilization; 2. To improve the resource utilization in production clusters with online/offline colocation.         |
+| [meiyapico](https://www.300188.cn/)       | [@complone](https://github.com/complone)                                             | sample   | Koordinatior is used as the elastic scheduler of flink, hudi, and alloxio to maximize the use of resources by using resource quotas and fair scheduling                                                                                                                                 |
```
Comment (jasonliu747) [inline ADOPTERS.md:21]: sample -> Sample

**Review by jasonliu747 — COMMENTED** [review]
Thanks for using Koordinator!

**Comment by hormes** [comment]
Welcome to the Koordinator community!

**Comment by FillZpp** [comment]
Welcome!

File: ADOPTERS.md (hunk 1 above)
Comment (complone) [reply ADOPTERS.md]: solved

**Pull request merged**